isis/src/base/objs/BulletDskShape/BulletDskShape.cpp

```diff
@@ -125,12 +133,12 @@ namespace Isis {
    *                        vertex. The vertices are ordered counter-clockwise
    *                        around the surface normal of the triangle.
    */
-  btMatrix3x3 BulletDskShape::getTriangle(const int index) const {
+  btMatrix3x3 BulletDskShape::getTriangle(const int index, const int segment) const {
```

Comment:
Where is this being called from? Does that code need to know which segment now?

Reply:
only `getnormal` it seems I updated it but it seems `getnormal` isn't used anywhere?

Reply:
Yeah, I don't know why getnormal is here? The bullet intersection routines already return the normal themselves.